base/collect_database.go

```diff
@@ -46,7 +46,7 @@ type Params struct {
 }
 
 const (
-	startingTimeStr = "StartingTime"
+	startingTimeStr = "startingTime"
 	elapsedTimeStr  = "ElapsedTime"
```

Comment:
@sriumcp Should `ElapsedTime` also be renamed to `elapsedTime` as well?

Reply:
That seems like a good idea ...